src/main/java/org/jitsi/videobridge/simulcast/SimulcastSenderManager.java

```diff
@@ -134,21 +134,49 @@ public boolean accept(RawPacket pkt)
     private synchronized SimulcastSender getOrCreateSimulcastSender(
         SimulcastReceiver simulcastReceiver)
     {
-        if (simulcastReceiver == null
-                || !simulcastReceiver.isSimulcastSignaled())
+        if (simulcastReceiver == null)
+        {
+            return null;
+        }
+
+        SimulcastStream[] simulcastStreams
+            = simulcastReceiver.getSimulcastStreams();
+
+        if (simulcastStreams == null || simulcastStreams.length == 0)
         {
+            // This is equivalent to !simulcastReceiver.isSimulcastSignaled() in
```

Comment:
I don't want you to modify this any further, I merely want to throw in my brief observations:

You're in a private method which is called by a single (at the time of this writing) other method. The caller has the check:

(simulcastReceiver == null || !simulcastReceiver.isSimulcastSignaled())

And the caller appears to perform the check just because it wants to call this method, the caller doesn't need simulcastReceiver itself.

But then you have the same checks in this method:
(1) (simulcastReceiver == null)
(2) Another, more verbose alternative to !simulcastReceiver.isSimulcastSignaled().

Again, I don't want to modify this PR any further and I merely wanted to state that I think there's repetition which adds complexity and, consequently, increases maintenance costs.